nixos/modules/services/misc/ollama.nix

```
@@ -9,6 +9,10 @@ in {
       enable = lib.mkEnableOption (
         lib.mdDoc "Server for local large language models"
       );
+      listenAddress = lib.mkOption {
+        type = lib.types.str;
+        default = "127.0.0.1:11434";
```

this could be added as well
```suggestion
        description = lib.mdDoc ''
          ...
        '';
```